presto-docs/src/main/sphinx/release/release-0.222.rst

```diff
@@ -0,0 +1,45 @@
+=============
+Release 0.222
+=============
+
+General Changes
+---------------
+* Fix incorrect results when dividing certain decimal numbers.
+* Fix query failure for certain join queries with "Equi criteria are empty,
+  so INNER join should not have PARTITIONED distribution type".
+* Add support for recoverable grouped execution. This can be enabled with the
```

Comment:
Can we link to the issue in case someone wants more details?

Reply:
@shixuan-fan can you point me to the issue ?

Reply:
Related issue: https://github.com/prestodb/presto/issues/12124